assets/sass/_variables.scss

```diff
@@ -0,0 +1,101 @@
+/*
+* Material Design colors!
+*/
+
+// main colors
+$color-blue:      #026ed2;
+$color-red:       #f44336;
+$color-green:     #4caf50;
+$color-pink:      #e91e63;
+$color-purple:    #9c27b0;
+$color-orange:    #ff9800;
+$color-dark-gray: #2b2b2b;
+
+// put into array for later use
+$main-colors: (
+  'default'   $color-blue,
+  'blue'      $color-blue,
+  'red'       $color-red,
```

Comment:
Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.